E2-Breakdown-Money/src/main/java/com/redelles/e2/Utils.java

```diff
@@ -4,20 +4,71 @@
 
 public class Utils {
 
-    public static Integer getNumber(){
+    public static Integer getAmount(){
+
+        Integer amount = -1;
+        Boolean flag;
+
+        do{
+            try {
+                System.out.println("Please enter an amount of money.");
+                amount = Utils.scanNumber();
+                flag = false;
+            } catch (Exception e) {
+                System.out.println("REMEMBER This application only works with numbers.");
+                flag = true;
+            }
+
+        } while (flag);
+
+        return amount;
+    }
+
+    public static Integer scanNumber(){
+
         Integer numberInput = 0;
         Scanner scan = new Scanner(System.in);
-
         numberInput = scan.nextInt();
 
         return numberInput;
     }
 
-    public static Integer breakdown(Integer qty, Integer result){
+    public static Integer breakdown(Integer quantity, Integer result){
```

Comment:
Breakdown it's a functionality which is too different of the rest of the  methods of this class.
You should extract this method to put in other place, a new class for example.

Also, inside the breakdown functionality not should be a display function inside him, this display function is like your view in your projects, therefore, it should be separately of this method.

This method should return  an array which will be displayed in other place